distribution/client/test/downloader.test.js

```diff
@@ -50,21 +49,21 @@ describe('the Downloader class', () => {
       });
 
       it('should manage to download a decently big file, retrying if needed', async () => {
-        // ace-editor is 5 MB, so it could make tests more flaky and cumbersome with slow connection
-        const toDownload = 'http://updates.jenkins-ci.org/download/plugins/ace-editor/1.1/ace-editor.hpi';
-        const sha256 = 'abc97028893c8a71581a5f559ea48e8e1f1a65164faee96dabfed9e95e9abad2';
+        // plugin is ~1 MB, so it could make tests more flaky and cumbersome with flaky connection
+        const toDownload = 'http://updates.jenkins-ci.org/download/plugins/blueocean-pipeline-editor/1.5.0/blueocean-pipeline-editor.hpi';
+        const sha256 = 'ccaaae7c899b7b15a5bd60f1e59336ca9adf5d08c01de5d123416e83ea5314db';
         await Downloader.download(toDownload, dir, 'ace-editor.hpi');
         expect(Checksum.signatureFromFile(`${dir}/ace-editor.hpi`)).toEqual(sha256);
       });
 
       it('should not retry too fast on failed download', async () => {
-        const toDownload = 'http://nonexisting-url-yada.org/thefile';
+        const toDownload = 'http://nonexisting-url-dont-retry-too-fast.org/thefile';
         const startTime = new Date();
         try {
-          await Downloader.download(toDownload, dir, 'thefile', {retry: 4, factor: 1.1, delay: 200});
+          await Downloader.download(toDownload, dir, 'thefile', null, {delay: 20, retry: 4, factor: 10});
           expect(false).toBeTruthy(); // fail(), should not reach this line.
         } catch (e) {
-          expect(new Date() - startTime).toBeGreaterThan(4 * 200);
+          expect(new Date() - startTime).toBeGreaterThan(0 + 20 + 200 + 2000);
```

Comment:
This little equation merits a code comment explaining what this magic number is for, I'm a bit confused looking at it myself.

Reply:
I agree on needing the explanation for this math, also the reason for the values on delay, retry, and factor would be awesome too.